gammapy/image/utils.py

```diff
@@ -338,10 +359,14 @@ def coordinates(image, world=True, lon_sym=True, radians=False, system=None):
     ----------
     image : `~astropy.io.fits.ImageHDU` or ndarray
         Input image
-    world : bool
+    world : bool, optional
         Use world coordinates (or pixel coordinates)?
-    lon_sym : bool
+    lon_sym : bool, optional
         Use symmetric longitude range ``(-180, 180)`` (or ``(0, 360)``)?
+    radians : bool, optional
+	Return coordinates in radians or degrees?
+    system : (unused!), optional
```

Comment:
When you notice something like this (`system` is unused), please either fix it or leave a line
```
TODO: `system` parameter is currently unused. Do we want it?
```
with a TODO marker we can easily search for, so that code cleanup becomes easier in the future.

In this case it would be OK if you remove the argument.